### ISSUE TITLE
Question on document.removeEventListener

### ISSUE THREAD

**Issue by dmrzn** [lgarron/clipboard.js#40]
Hi Lucas
I see there are document.addEventListener calls in clipboard.js. Shouldn't be there document.removeEventListener calls to cleanup?

**Comment by lgarron** [comment]
The event listeners are added (once) to `document` for its lifetime, while `cleanup()` runs per invocation of `clipboard.copy()`. This is a bit hard to follow in the code, because of the workarounds we have to do.

It's possible to support removing the event listeners from `document`, but no one has asked for that.

**Comment by dmrzn** [comment]
Ok, I now see that _intercept variable is set to true or false to avoid event handling when it's not needed.
